apps/mobile/utils/bridgeHandler.ts

```diff
@@ -0,0 +1,53 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { WEBVIEW_MESSAGE_TYPES } from "@repo/constants";
+import { RefObject } from "react";
+import { Platform } from "react-native";
+import WebView from "react-native-webview";
+import { WebViewProgressEvent, WebViewMessageEvent } from "react-native-webview/lib/WebViewTypes";
+
+import webviewLoginBridge from "@/utils/webviewLoginBridge";
+
+import { parseMessage } from "./parseMessage";
+import { sendMessageToWeb } from "./sendMessageToWeb";
+
+const isIos = Platform.OS === "ios";
+const isAndroid = Platform.OS === "android";
+
+const sendData = async (webViewRef: RefObject<WebView<object>>) => {
+  const accessToken = await AsyncStorage.getItem("accessToken");
+  const userStr = await AsyncStorage.getItem("user");
+
+  const sendMessageData = {
```

Comment:
```suggestion
  const userData = {
```
send라는 행위를 하는거같지 않아서 굳이 수정한다면 userData정도로 수정해도 좋을듯?